ci-operator/config/openshift/cluster-api/openshift-cluster-api-openshift-master.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
```

Comment:
The file name has an extra "openshift", it should be `openshift-cluster-api-master.yaml`

Reply:
thanks, I copied the filename from 4.2 release yaml